### PR TITLE
Stdout capture

### PR DESCRIPTION
**Pull request by duylethanh**

### CODE REVIEW

File: simexpal/launch/common.py [hunk 1]
```diff
@@ -218,12 +218,19 @@ def compile_manifest(run):
...
-	with open(manifest.output_file_path('out'), "w") as f:
-		# We do not actually need to write anything to the output file.
-		# However, we might want to pipe experimental output to it.
-		if manifest.output == 'stdout':
+	(stdout_pipe, stdout) = (None, None)
+	if manifest.output == 'stdout':
+		with open(manifest.output_file_path('out'), "w") as f:
+			# We do not actually need to write anything to the output file.
+			# However, we might want to pipe experimental output to it.
 			stdout = os.dup(f.fileno())
```
Comment (avdgrinten) [inline simexpal/launch/common.py]: We should still create the output file even if `output != 'stdout'`. In that case, however, we don't need to `dup` the file handle.

File: examples/sorting/experiments.yml [hunk 2]
```diff
@@ -15,7 +15,5 @@ instances:
 experiments:
   - name: insertion-sort
     args: ['./sort.py', '--algo=insertion-sort', '@INSTANCE@']
-    output: stdout
```
Comment (avdgrinten) [inline examples/sorting/experiments.yml]: We should not change the example here (its output is needed to make an evaluation script work).

Comment (duylethanh) [reply examples/sorting/experiments.yml]: I reverted the changes to the `experiment.yml`.

File: simexpal/launch/common.py (hunk 1 above)
Comment (duylethanh) [reply simexpal/launch/common.py]: Done. The check `manifest.output == 'stdout'` is done after creating the output file.

**Pull request merged**